test/app.test.js

```diff
@@ -335,6 +337,30 @@ describe('app routes', function () {
             const account = await accountService.getAccount(accountId);
             expect(account).ok;
         });
+
+        it('returns 403 if named account does not exist', async () => {
+            const nonExistentNamedAccountId = testAccountHelper.buildTestAccountId();
```

Comment:
This is essentially a UUID so there is a (low?) chance of it actually existing on-chain and this test therefore failing. This is the same risk we take when creating new accounts via `testAccountHelper.createNEARAccount()` though so is potentially fine?